### PR TITLE
Updated README & removed redundant code related to env vars

### LINKED ISSUE #854: Verify README is up to date
Run a fresh install and see if anything in README needs to be updated.

### PR DESCRIPTION
**Pull request by mmmavis**
Fixes #854

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
     "copy:index": "shx cp index.html dist/index.html",
     "copy:manifest": "shx cp manifest.json dist",
     "copy:contributejson": "shx cp contribute.json dist",
-    "copy:assets": "shx cp favicon.png dist && shx cp -r assets dist",
+    "copy:assets": "shx cp -r assets dist",
```
Comment (mmmavis) [inline package.json]: `favicon.png` isn't used anywhere

File: js/utility.js [hunk 2]
```diff
@@ -21,14 +21,11 @@ const Utils = {
     return out;
   },
 
-  getCurrentURL(router=false) {
+  getCurrentURL() {
```
Comment (mmmavis) [inline js/utility.js]: `getCurrentURL()` is never used with an `argument` passed in. I guess this is just some old code that we forgot to update.

File: js/utility.js [hunk 3]
```diff
@@ -21,14 +21,11 @@ const Utils = {
...
     if (typeof window !== `undefined`) {
       return window.location.toString();
     }
-    if (router) {
-      return `${env.ORIGIN}${router.getCurrentLocation().pathname}`;
```
Comment (mmmavis) [inline js/utility.js]: Removing this chunk meaning `ORIGIN` env is no longer needed.

File: js/build-env-config.js [hunk 4]
```diff
@@ -12,14 +12,10 @@ if (environment !== `PRODUCTION`) {
   habitat.load(`config/default.env`);
 }
 
-const HOST = habitat.get(`HOST`);
 const PORT = habitat.get(`PORT`);
 const LEARN_MORE_LINK = habitat.get(`LEARN_MORE_LINK`);
-const ORIGIN = HOST + (PORT ? `:${PORT}` : ``);
```
Comment (mmmavis) [inline js/build-env-config.js]: `ORIGIN` not being used anywhere now. We don't need it... and hence no need to have `HOST`.

File: README.md [hunk 5]
```diff
@@ -1,14 +1,13 @@
-[![Travis Build Status](https://travis-ci.org/mozilla/network-pulse.svg?branch=master)](https://travis-ci.org/mozilla/network-pulse) [![AppVeyor Build Status](https://ci.appveyor.com/api/projects/status/github/mozilla/network-pulse?svg=true)](https://ci.appveyor.com/project/mozillafoundation/network-pulse) [![Dependency Status](https://david-dm.org/mozilla/network-pulse.svg)](https://david-dm.org/mozilla/network-pulse) [![Dev Dependency Status](https://david-dm.org/mozilla/network-pulse/dev-status.svg)](https://david-dm.org/mozilla/network-pulse/?type=dev)
-
 # Mozilla Pulse
 
+[![Travis Build Status](https://travis-ci.org/mozilla/network-pulse.svg?branch=master)](https://travis-ci.org/mozilla/network-pulse) [![AppVeyor Build Status](https://ci.appveyor.com/api/projects/status/github/mozilla/network-pulse?svg=true)](https://ci.appveyor.com/project/mozillafoundation/network-pulse) [![Dependency Status](https://david-dm.org/mozilla/network-pulse.svg)](https://david-dm.org/mozilla/network-pulse) [![Dev Dependency Status](https://david-dm.org/mozilla/network-pulse/dev-status.svg)](https://david-dm.org/mozilla/network-pulse/?type=dev)
```
Comment (mmmavis) [inline README.md]: I just moved this line down below the big heading

File: README.md [hunk 6]
```diff
@@ -19,66 +18,52 @@ Interested in contributing to this project? A good place to start is by helping
 
 Browse the [issues](https://github.com/mozilla/network-pulse/issues). Look for the labels [enhancement](https://github.com/mozilla/network-pulse/labels/enhancement), [help wanted](https://github.com/mozilla/network-pulse/labels/help%20wanted), [good first issue](https://github.com/mozilla/network-pulse/labels/good%20first%20issue).
 
-
 ## Development
```
Comment (alanmoo) [inline README.md]: This section is now empty? What's it for?

**Pull request merged**